aui.boot.cmake

```diff
@@ -116,8 +116,13 @@ else()
     set(AUI_TARGET_ABI "${_tmp}" CACHE INTERNAL "COMPILER-PROCESSOR pair")
 endif()
 
+# checking if custom cache dir is set for the system
+if(DEFINED ENV{AUIB_CACHE_DIR})
+    string(REPLACE "\\" "/" _tmp $ENV{AUIB_CACHE_DIR}) # little hack to handle Windows paths
+else()
+    set(_tmp ${HOME_DIR}/.aui)
+endif()
 
-set(_tmp ${HOME_DIR}/.aui)
 if(AUIB_LOCAL_CACHE)
     set(_tmp ${CMAKE_BINARY_DIR}/aui.boot)
 endif()
```

Comment:
Have you tried to use AUI_CACHE_DIR? (see declaration on line 130)

https://aui-framework.github.io/master/md_docs_AUI_configure_flags.html

Reply:
That's a good catch, no I did not tried it and it will surely work. But since it's a per project config I think it may still be helpful to have a way to configure the cache dir once for every AUI projects